nilearn/reporting/_get_clusters_table.py

```diff
@@ -25,11 +25,11 @@ def _local_max(data, affine, min_distance):
     data : array_like
         3D array of with masked values for cluster.
 
-    affine: np.ndarray
+    affine : np.ndarray
         Square matrix specifying the position of the image array data
         in a reference space.
 
-    min_distance : `int`
+    min_distance : int
```

Comment:
WDYT about using the ``` :obj:`int` ``` pattern? That way it can link to the object definition automatically.

Reply:
@tsalo I think that we should do that at some point since it's a clear enhancement, but I also agree with @bthirion that this PR is already huge. I realize now that I should have broken it in separate smaller PRs since, even if I'm only modifying docstrings, it will be a pain to review.